File: Backend/maintenace-service/src/Entity/Patio.cs
namespace Entity
{
    public class Patio
    {
        public string Id { get; set; }              // Identificador único del patio
        public string IdBodega { get; set; }        // FK de la bodega asociada
        public string Nombre { get; set; }          // Nombre del patio
        public bool Estado { get; set; }            // Estado (activo/inactivo)
        public DateTime Fecha_log { get; set; }     // Fecha de creación o última modificación
    }
}

File: Frontend/src/Types/patio.ts
export interface Patio {
  id: string;
  idBodega: string;
  nombre: string;
  estado: boolean;
  fecha_log: string;
}

export interface PatioResponse {
  data: Patio[];
  message: string;
  success: boolean;
}

export interface PatioCreateRequest {
  idBodega: string;
  nombre: string;
  estado: boolean;
  fecha_log: string;
}

export interface PatioUpdateRequest extends PatioCreateRequest {
  id: string;
}
